src/net/body.rs

```diff
@@ -357,12 +357,12 @@ enum FileField_ {
 }
 
 trait StreamField: Read + Send + 'static {
-    fn add_self(self: Box<Self>, name: String, filename: Option<String>, content_type: Option<Mime>, to: &mut Multipart);
+    fn add_self(self: Self, name: String, filename: Option<String>, content_type: Option<Mime>, to: &mut Multipart);
```

Comment:
Sorry for not commenting on that earlier, I'm a little bit out of my depth with here to be honest.